analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -416,6 +309,115 @@ open class Npm(
         return modulesDir.takeIf { it.name == "node_modules" }
     }
 
+    /**
+     * Construct a [Package] by parsing its _package.json_ file and - if applicable - querying additional
+     * content via the `npm view` command. The result is a [Pair] with the raw identifier and the new package.
+     */
+    @Suppress("HttpUrlsUsage")
+    internal fun parsePackage(packageFile: File): Pair<String, Package> {
```

Comment:
Is this function just being moved or also modified?

Would it make sense to factor out moving of code to a separate commit for easier review?

Reply:
It was moved out of static context, but in the same run needed to be modified (code was factored out to ``) to do the right thing for Yarn. I considered splitting the move from the modifications, but could not figure out how in this case (without leaving things intermediately in a broken state).

Reply:
Ok, I found a way to improve this.

Reply:
Thanks!